toolchest_client/api/query.py

```diff
@@ -217,7 +217,7 @@ def _register_input_file(self, input_file_path, input_prefix, input_order):
             'input-files'
         ])
         file_name = os.path.basename(input_file_path)
-        if "?" in file_name:
+        if "?" in file_name and "http" in input_file_path:
```

Comment:
Is it possible to move this a few lines down, and use `input_is_http_url`? `"http" in input_file_path` catches files outside of HTTP urls (e.g. "./fastq_1_from_ncbi_from_http.fastq")